examples/simple-jsbeautifyrc/blade/original/_test.blade.php

```diff
@@ -0,0 +1,31 @@
+<html>
```

Comment:
This test is disabled because of the prefix `_` in `_test.blade.php`.

Please enable this test so I can confirm this new feature works.